src/decorators/EsField.ts

```diff
@@ -0,0 +1,5 @@
+import { Field } from '../models/Mapping';
+import { generatePropertyDecorator } from '../utils/decorators';
+
+// tslint:disable-next-line:variable-name - This is a decorator
```

Comment:
IMO this isn't required anymore once you upgrade `@smartive/tslint-config` to v3

Reply:
We can have a look at this matter. The v3 of tslint config does allow it on `function-names` not on constants. So if it's an actual function, it would work (instead of a const)